src/main/scala/com/lynbrookrobotics/sbt/Tasks.scala

```diff
@@ -218,6 +220,23 @@ object Tasks {
     }
   }
 
+  lazy val robotConsole: Def.Initialize[Task[Unit]] = Def.task {
+
+    val SIZE = 1024
+    val PORT = 6666
+    val logger = streams.value.log
+
+    val socket = new DatagramSocket(PORT)
+    while (true) {
+      val buffer = new Array[Byte](SIZE)
+      val packet = new DatagramPacket(buffer, buffer.length)
+      socket.receive(packet)
+      val message = new String(packet.getData)
+      val ipAddress = packet.getAddress().toString
+      printf(message)
```

Comment:
Is there any reason you are using `printf` instead of just `print`?
I supposed the \n is included in the `DatagramPacket`.

Reply:
It seems like I could crash this program by running `println("%d")` on the RoboRIO.

Reply:
i don't expect any adverse effects from passing format specifiers without values
it will simply print the string

but if we're removing `ipAddress` we might as well change this to `print` though